apps/namadillo/src/atoms/integrations/services.ts

```diff
@@ -52,9 +52,14 @@ const getShieldedArgs = async (
   };
 };
 
-export type AssetWithBalance = {
+export type AssetWithBalanceAndIbc = {
```

Comment:
I would maybe rename this to `AssetWithBalanceAndIbcDetails` or `AssetWithBalanceAndIbcInfo` or something that indicates what the `Ibc` part is :)

Reply:
Now it's possible to compose the definition of `AssetWithBalance` from `types.d.ts`

Reply:
Sounds good! I will rename it.